### DISCUSSION TITLE
How do I fix the curl certificate expiry problem?

### DISCUSSION THREAD

**Discussion by cryo-kronos** [Homebrew/discussions#2234] [Everyday usage] [answered]
When I run "brew upgrade", I now get a SSL certificate expired message with curl. How do I update the certificate? I'm on macOS Mojave.

> curl: (60) SSL certificate problem: certificate has expired
More details here: https://curl.haxx.se/docs/sslcerts.html

> curl performs SSL certificate verification by default, using a "bundle"
 of Certificate Authority (CA) public keys (CA certs). If the default
 bundle file isn't adequate, you can specify an alternate file
 using the --cacert option.
> If this HTTPS server uses a certificate signed by a CA represented in
 the bundle, the certificate verification probably failed due to a
 problem with the certificate (it might be expired, or the name might
 not match the domain name in the URL).
>If you'd like to turn off curl's verification of the certificate, use
 the -k (or --insecure) option.
HTTPS-proxy has similar options --proxy-cacert and --proxy-insecure.

**Comment by Bo98** [comment, marked as answer]
This is an issue with macOS's `curl`, which we're investigating and working on a fix. A workaround for now is to set `HOMEBREW_FORCE_BREWED_CURL=1` which will make `brew` use our build of `curl` instead.

**Reply by cryo-kronos** [reply]
Thanks for the update and the workaround.

I did some more searching and found an answer on Stackoverflow which seems to have fixed the issue for me, although it requires manually editing the certs.pem file in /etc/ssl to remove the expired DST Root CA X3 certificate snippet from certs.pem.

https://stackoverflow.com/a/69396425/2506813
